project/FrontEnd/collaborative_science_platform/lib/screens/profile_page/widgets/about_me.dart

```diff
@@ -1,21 +1,40 @@
 import 'package:collaborative_science_platform/utils/colors.dart';
 import 'package:collaborative_science_platform/utils/responsive/responsive.dart';
+import 'package:collaborative_science_platform/widgets/app_button.dart';
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 
-class AboutMe extends StatelessWidget {
+class AboutMe extends StatefulWidget {
   final String email;
   final String name;
   final String surname;
   final int noWorks;
   final String aboutMe;
+  final bool isBanned;
```

Comment:
woah pretty complicated and scary constructor. There are always better ways to pass these kind of informations. Just keep in mind not to do that in the future, but for this class as it is used by just one parent widget, that's OK